libkbfs/folder_update_prepper.go

```diff
@@ -331,6 +331,13 @@ type pathTreeNode struct {
 	mergedPath path
 }
 
+type prepFolderFlags byte
+
+const (
+	// The blocks of indirect files should be copied.
+	prepFolderCopyIndirectFileBlocks prepFolderFlags = 1 << iota
```

Comment:
Was undecided about making this a bool or a flag.  For a while I had a second option that I got rid of, but I figure we might end up needing more options once dir updates are involved, so I think flags are probably the way to go.